packages/qwik-city/src/buildtime/build-pages-rewrited.unit.ts

```diff
@@ -228,3 +228,33 @@ testWithDuplicatedRoutes(
     assert.equal(r.pathname, '/produkt/');
   }
 );
+
+const testWithDuplicate = testAppSuite('ssss', {
+  rewriteRoutes: [
+    {
+      prefix: undefined,
+      paths: {},
+    },
+    {
+      prefix: undefined,
+      paths: {
+        produkt: 'produkt',
+      },
+    },
+    {
+      prefix: undefined,
+      paths: {
+        booking: 'reservation',
+      },
+    },
+  ],
+});
+
+testWithDuplicate(
+  'Issue #6799: Bug while using rewrite routes pointing to the same file',
+  ({ assertRoute }) => {
+    const r = assertRoute('/produkt/');
+
+    assert.equal(r.pathname, '/produkt/');
```

Comment:
- can you rename this to `testWithAlias`
- I don't understand how these are pointing to the same route? Does this fail without the change in build.ts?